### DISCUSSION TITLE
Enotempty error when trying to update plug in

### DISCUSSION THREAD

**Discussion by thrangster** [dgreif/ring#937] [General]
I am trying to update to the Lester’s version of the plug in and am getting this error…

![689B21C1-495D-4EA0-B9D2-E3E47A9EF638](https://user-images.githubusercontent.com/68309317/168209630-3d20b708-f983-410c-a04b-e1f520f0b65d.jpeg)

I cannot delete the plug in, I get a similar error. I tried an rm -rf of both the path and destination in terminal, but nothing happens… latest node installed…

how can I rectify this? Thanks

**Comment by dgreif** [comment]
See https://github.com/dgreif/ring/issues/943#issuecomment-1130254819
